### PR TITLE
python-vint: update to 0.3.13

### PR DESCRIPTION
**Pull request by lafrenierejm**
Minor changes were made to $short_desc to normalize it with other Python
packages.  $distfiles now uses PyPi rather than the project's GitHub
releases.

### CODE REVIEW

File: srcpkgs/python-vint/template [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # Template file for 'python-vint'
 pkgname=python-vint
-version=0.3.11
-revision=2
+version=0.3.13
+revision=3
```
Comment (Vaelatern) [inline srcpkgs/python-vint/template]: `revision=1`

File: srcpkgs/python-vint/template [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 noarch=yes
-wrksrc="vint-${version}"
+wrksrc="vim-vint-${version}"
 build_style=python-module
 pycompile_module="vint"
 hostmakedepends="python-setuptools python3-setuptools"
-depends="python-ansicolor python-chardet python-enum34 python-pathlib python-yaml"
-short_desc="Python2 lint tool for Vim script Language"
+depends="python-ansicolor python-chardet python-enum34 python-pathlib
+ python-yaml-3.11_2"
```
Comment (asergi) [inline srcpkgs/python-vint/template]: `python-yaml`

File: srcpkgs/python-vint/template [hunk 3]
```diff
@@ -22,7 +23,7 @@ post_install() {
 
 python3-vint_package() {
 	noarch=yes
-	depends="python3-ansicolor python3-chardet python3-yaml"
+	depends="python3-ansicolor python3-chardet python3-yaml-3.11_2"
```
Comment (asergi) [inline srcpkgs/python-vint/template]: `python3-yaml`

**Comment by Duncaen** [comment]
Merged as cd7d1b03d77642c33e5e5f237186bc4098a94c03.

**Pull request closed without merging**